cves/CVE-2013-0848.yml

```diff
@@ -173,10 +180,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
+  answer: This vulnerability was found by two developers.
   automated:
   contest:
-  developer:
+  developer: Mateusz Jurczyk and Gynvael Coldwind
```

Comment:
I think `developer` is supposed to be set as `true`, and the names can be moved to the `answer` field. Also out of curiosity, how did you find the people who found it? I found it hard to find any direct attribution for my CVEs so explaining your process in your writeup would be interesting.